### PR TITLE
update gas fees

### PR DESCRIPTION
**Pull request by aliensarefake**
https://www.notion.so/switcheo/Replace-keplr-chain-registry-with-cosmos-chain-registry-219a38df951b4ae296d0c8e5400d0885?pvs=4

### CODE REVIEW

**Comment by HeesungB** [comment]
@aliensarefake I can't see notion page.
Can you explain about it?

File: cosmos/carbon.json [hunk 1]
```diff
@@ -110,9 +110,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "binancecoin",
       "gasPriceStep": {
-        "low": 769230769.2307693,
-        "average": 961538461.5384616,
-        "high": 1153846153.8461537
+        "low": 1000000,
+        "average": 1000000,
+        "high": 11000000
```
Comment (GulfOfAlaska) [inline cosmos/carbon.json:115]: seems to be 100000? can doublecheck all values again?
image

**Review by GulfOfAlaska — APPROVED** [review]

File: cosmos/carbon.json [hunk 2]
```diff
@@ -99,9 +99,9 @@
       "coinDecimals": 8,
       "coinGeckoId": "switcheo",
       "gasPriceStep": {
-        "low": 769.2307692307693,
-        "average": 961.5384615384615,
-        "high": 1153.8461538461538
+        "low": 1,
+        "average": 1,
+        "high": 1
```
Comment (sarah-thong) [inline cosmos/carbon.json:104]: Not sure if this is correct cos the gas fee calculation is as follows
`Gas Fee = gasPriceStep * gasCost`

In the case of low price step,
Gas Fee = 769.2307692307693 * 300000 = 2 SWTH (more than enough fees for standard transaction)

By changing `769.2307692307693` to `1`, sending tokens directly via Keplr form might not work anymore.
Screenshot 2024-01-19 at 5 00 29 PM

i.e. if gas price step = 1,
Gas Fee = 1 * 300000 = 0.003 SWTH (not enough fees paid to send tokens => tx failed).

Think what would be better to recalculate the `low` value (such that tx fee = 1 SWTH), then apply the same value to `average` and `high`.

Calculation for `low` is as follows:
```typescript
gasPriceStep = (1 * 10^8) / 300000 (13 d.p. rounded up)
```
where SWTH decimals = 8 and Gas Cost on Keplr = 300000

File: cosmos/carbon.json [hunk 3]
```diff
@@ -326,9 +326,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "usd-coin",
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:331]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.

File: cosmos/carbon.json [hunk 4]
```diff
@@ -315,9 +315,9 @@
       "coinDecimals": 6,
       "coinGeckoId": "usd-coin",
       "gasPriceStep": {
-        "low": 0.07692307692307693,
-        "average": 0.09615384615384616,
-        "high": 0.11538461538461539
+        "low": 0.033333333333,
+        "average": 0.033333333333,
+        "high": 0.033333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:320]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.

File: cosmos/carbon.json [hunk 5]
```diff
@@ -304,9 +304,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "usd-coin",
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:309]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.

File: cosmos/carbon.json [hunk 6]
```diff
@@ -283,19 +283,19 @@
...
       "coinDenom": "USC",
       "coinMinimalDenom": "usc",
       "coinDecimals": 6,
       "gasPriceStep": {
-        "low": 0.07692307692307693,
-        "average": 0.09615384615384616,
-        "high": 0.11538461538461539
+        "low": 0.033333333333,
+        "average": 0.033333333333,
+        "high": 0.033333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:298]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.

File: cosmos/carbon.json [hunk 7]
```diff
@@ -262,19 +262,19 @@
       "coinDecimals": 6,
       "coinGeckoId": "cosmos",
       "gasPriceStep": {
-        "low": 0.007692307692307693,
-        "average": 0.009615384615384616,
-        "high": 0.011538461538461539
+        "low": 0.003333333333,
+        "average": 0.003333333333,
+        "high": 0.003333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:267]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.

File: cosmos/carbon.json [hunk 8]
```diff
@@ -262,19 +262,19 @@
...
       "coinDenom": "stATOM",
       "coinMinimalDenom": "ibc/B7864B03E1B9FD4F049243E92ABD691586F682137037A9F3FCA5222815620B3C",
       "coinDecimals": 6,
       "gasPriceStep": {
-        "low": 0.007692307692307693,
-        "average": 0.009615384615384616,
-        "high": 0.011538461538461539
+        "low": 0.003333333333,
+        "average": 0.003333333333,
+        "high": 0.003333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:277]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.

File: cosmos/carbon.json [hunk 9]
```diff
@@ -251,9 +251,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "canto",
       "gasPriceStep": {
-        "low": 153846153846.15384,
-        "average": 192307692307.6923,
-        "high": 230769230769.23077
+        "low": 66666666666.666666666667,
+        "average": 66666666666.666666666667,
+        "high": 66666666666.666666666667
```
Comment (sarah-thong) [inline cosmos/carbon.json:256]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.

File: cosmos/carbon.json [hunk 10]
```diff
@@ -164,9 +164,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "ethereum",
       "gasPriceStep": {
-        "low": 76923076.92307693,
-        "average": 96153846.15384616,
-        "high": 115384615.38461539
+        "low": 33333333.333333333333,
+        "average": 33333333.333333333333,
+        "high": 33333333.333333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:169]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 8 d.p.

File: cosmos/carbon.json [hunk 11]
```diff
@@ -153,9 +153,9 @@
...
```
Comment (sarah-thong) [inline cosmos/carbon.json:158]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 8 d.p.

File: cosmos/carbon.json [hunk 12]
```diff
@@ -132,19 +132,19 @@
...
       "coinDenom": "CGLP",
       "coinMinimalDenom": "cglp.1.19.1698d3",
       "coinDecimals": 18,
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:147]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.

File: cosmos/carbon.json [hunk 13]
```diff
@@ -132,19 +132,19 @@
       "coinDecimals": 18,
       "coinGeckoId": "binance-usd",
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```
Comment (sarah-thong) [inline cosmos/carbon.json:137]: Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.

**Comment by HeesungB** [comment]
@aliensarefake @sarah-thong @GulfOfAlaska Let me know when you're ready!

**Comment by sarah-thong** [comment]
Hi @HeesungB, thanks for your patience! We're ready to merge already.

**Pull request merged**